src/storage/PassthroughStore.ts

```diff
@@ -0,0 +1,41 @@
+import { Conditions } from './Conditions';
+import { Patch } from '../ldp/http/Patch';
+import { Representation } from '../ldp/representation/Representation';
+import { RepresentationPreferences } from '../ldp/representation/RepresentationPreferences';
+import { ResourceIdentifier } from '../ldp/representation/ResourceIdentifier';
+import { ResourceStore } from './ResourceStore';
+
+/**
+ * Store that calls the corresponding functions of the source Store.
+ * Can be extended by stores that do not want to override all functions.
+ */
```

Comment:
Perhaps just explicitly mention the "decorator" pattern in the comment, which is the name of what this does.